config/examples/imx-rt1060.config

```diff
@@ -0,0 +1,29 @@
+ARCH?=ARM
+TARGET?=imx_rt
+SIGN?=ECC256
+HASH?=SHA256
+MCUXPRESSO?=$(PWD)/../SDK-2.8.2_EVK-MIMXRT1060
```

Comment:
It might be obvious to some, but can you add a link / note in the Targets.md for where to download the MCU SDK and what components are required. I used their SDK building for the `EVK-MIMXRT1060` and default components selected "Components: FreeRTOS, sdmmc stack, mflash". Then extracted next to wolfBoot-root and renamed to `SDK-2.8.2_EVK-MIMXRT1060`. All builds correctly, but cannot test (no hardware).